File: 251003/stasevich/java/com/lab1/lab1DC/exception/ErrorResponse.java
package com.lab1.lab1DC.exception;

public class ErrorResponse {
    private String errorCode;
    private String errorMessage;
    private int statusCode;

    public ErrorResponse(String errorCode, String errorMessage, int statusCode) {
        this.errorCode = errorCode;
        this.errorMessage = errorMessage;
        this.statusCode = statusCode;
    }

}

File: 351001/shevchenkoAD/DistComp/DistComp/Application/DTOs/Responses/ErrorResponse.cs
using System.Text.Json.Serialization;

namespace DistComp.Application.DTOs.Responses;

public record ErrorResponse {
    public ErrorResponse(string errorMessage,
                         int errorCode) {
        ErrorMessage = errorMessage;
        ErrorCode = errorCode;
    }

    public string ErrorMessage { get; init; }

    [JsonPropertyName("errorCode")] public int ErrorCode { get; init; }
}